graph/R-GAT/README.md

```diff
@@ -106,23 +122,17 @@ cd $GRAPH_FOLDER
 python3 tools/split_seeds.py --path igbh --dataset_size full
 ```
 
-**Compress graph (optional, only for GLT implementation)**
-```bash
-cd $GRAPH_FOLDER
-python3 tools/compress_graph.py --path igbh --dataset_size tiny --layout <CSC or CSR>
-```
 
 #### Calibration dataset
-TODO
+
+The calibration dataset contains 5000 nodes from the training paper nodes of the IGBH dataset. We provide the [Node ids](../../calibration/IGBH/calibration.txt) and the [script](tools/split_seeds.py) to generate them (using the `--calibration` flag). 
 
 
 ### Run the benchmark
 #### Debug Run
 ```bash
 # Go to the benchmark folder
 cd $GRAPH_FOLDER
-# Run the benchmark GLT
-python3 main.py --dataset igbh-glt-tiny --dataset-path igbh/ --profile debug-glt [--model-path <path_to_ckpt>] [--in-memory] [--device <cpu or gpu>] [--dtype <fp16 or fp32>] [--scenario <SingleStream, MultiStream, Server or Offline>] [--layout <COO, CSC or CSR>]
 
 # Run the benchmark DGL
 python3 main.py --dataset igbh-dgl-tiny --dataset-path igbh/ --profile debug-dgl [--model-path <path_to_ckpt>] [--in-memory] [--device <cpu or gpu>] [--dtype <fp16 or fp32>] [--scenario <SingleStream, MultiStream, Server or Offline>]
```

Comment:
For official submissions, only `Offline` is required and counts for official results, but all the scenarios are supported in the reference. Just in case someone wants run the other ones